src/Themes.js

```diff
@@ -1,5 +1,7 @@
 const Themes = {
   'Default' : {
+    icon: '#666',
+    iconSelected: '#ffbc00',
```

Comment:
I think the name is a little too generic, because there are a lot of icons in Reacticz but this only changes the app menu icons. What do you think of 'menuIcon' and 'menuIconSelected' instead ? Also could you please keep the properties in alphabetic order ?